DirectEmployers/MyJobs
# Changes needed for jasmine test reports.

## What's this PR do?
When we run gulp test, produce a test report in `Myjobs/reports/gulp_test.xml`

## Screenshots (if appropriate)

## Where should the reviewer start?
It's 5 lines.

## How should this be manually tested?
```
git checkout jasmine-test-reports
cd gulp
npm install
gulp test
```
confirm the existence of `Myjobs/reports/gulp_test.xml`.

## Any background context you want to provide?
The goal of this is to produce useful test information on jenkins, so before this is merged, make sure the jasmine tests show up in jenkins' test records.
